### DISCUSSION TITLE
Add Mosparo as a Captcha provider

### DISCUSSION THREAD

**Discussion by skid9000** [goauthentik/authentik#8838] [Ideas]
[Mosparo](https://mosparo.io/) is a FOSS, self hostable, and most importantly accessible captcha. Integration seems doable as an alternative to reCaptcha.

**Comment by GeorgeDaGreatt** [comment]
I love the idea of adding support for more Captcha providers as drop-in replacements for reCaptcha, CF Turnstile, etc. I would also recommend adding PoW (Proof of Work) Captchas like [Cap](https://capjs.js.org/), [Private Captcha](https://privatecaptcha.com/) and [Altcha ](https://altcha.org/)(Plus a couple more). 

These alternatives are:
- More Accessible, no puzzle-solving needed because verifying is done using complex mathematics in the background
- Fast, especially on newer devices
- Are all FOSS (excluding Altcha, which is paid)
- Easy to integrate

I hope support for these Captcha providers gets added along with Mosparo.
